packages/nimble-components/src/wafer-map/modules/matrix-renderer.ts

```diff
@@ -0,0 +1,17 @@
+import { wrap, Remote } from 'comlink';
+import type { WaferMap } from '..';
+import { workerCode } from '../workers/render-worker';
+import type { RenderWorker } from '../../../build/generate-workers/dist/esm/source/render-worker';
+
+/**
+ * Responsible for drawing the dies inside the wafer map, adding dieText and scaling the canvas
+ */
+export class MatrixRenderer {
+    public readonly workerOne: Remote<RenderWorker>;
+
+    public constructor(private readonly wafermap: WaferMap) {
+        const blob = new Blob([workerCode], { type: 'text/javascript' });
+        const url = URL.createObjectURL(blob);
+        this.workerOne = wrap<RenderWorker>(new Worker(url));
+    }
+}
```

Comment:
Instead of having this wrapper around the RenderWorker instance, I think with the above changes we can expose the RenderWorker class more directly instead of having to access the "workerOne" property.

```ts
export createMatrixRenderer = async (): Remote<RenderWorker> {
    const blob = new Blob([workerCode], { type: 'text/javascript' });
    const url = URL.createObjectURL(blob);
    const RemoteMatrixRenderer = wrap<RenderWorker>(new Worker(url));
    const instance = await new RemoteMatrixRenderer();
    return instance;
}

```

Reply:
good point, did it
![image](https://github.com/ni/nimble/assets/110180309/085aaa50-b21a-4273-823c-3f3a2fdccf29)